ui/src/Groups/Counter/Counter.spec.js

```diff
@@ -0,0 +1,68 @@
+import React from 'react';
+import { Counter } from './index.js';
+import { render } from 'TestUtils';
+
+describe('Counter Group component', () => {
+  const title = 'Test title here';
+  const subtitle = 'Test subtitle here';
+  const count = 123;
+  const testClassName = 'testCssClass';
+
+  const confirmHasContent = (contentExpected) => (content) =>
+    content === contentExpected.toString(); // has the expected content
+
+  const confirmHasClassName = (classNameExpected) => (content, node) =>
+    node.classList.contains('Counter') && // has correct Block class
+    node.classList.contains(classNameExpected); // has the expected classname
+
+  it('renders the expected component', () => {
+    const { getByText } = render(
+      <Counter title={title} subtitle={subtitle} count={count} />
+    );
+    expect(getByText(confirmHasContent(title))).toBeInTheDocument();
```

Comment:
I think for these string checks you could try just ` expect(getByText(subtitle)).toBeInTheDocument();` . The providing of a function as a matcher is needed if both content and the element the content is in need to be checked.

Reply:
Good point. Updated.